Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -251,9 +251,9 @@ def pull_request_branch_point(cls, repository, args, **kwargs):
             source_remote = repository.source_remotes()[-1]
             args.remote = source_remote
 
-        if repository.branch is None or repository.branch in repository.DEFAULT_BRANCHES or \
+        if not repository.dev_branches.match(repository.branch) and (repository.branch is None or repository.branch in repository.DEFAULT_BRANCHES or
                 repository.PROD_BRANCHES.match(repository.branch) or \
-                repository.branch in repository.branches_for(remote=source_remote):
+                repository.branch in repository.branches_for(remote=source_remote)):
```

Comment:
Logic looks fine, but have we tried to write a test for this? I think I've written a few `pull_request_branch_point` tests.

Reply:
Unfortunately no existing `pull_request_branch_point` tests. I looked into it and the overhead for writing tests for this is more than they're worth (need a lot of mocks and repo setup).